### PR TITLE
CHORES: Update the procedures for building Engine Extension

### PR DESCRIPTION
**Pull request by dixonsiu**

### CODE REVIEW

**Review by shimono — APPROVED** [review]

File: ja/server-operator/setup_engine_extensions.md [hunk 1]
```diff
@@ -10,9 +10,7 @@ Engine ExtensionはEngine Libraryの機能を拡張するための機構です
 * [personium-ex-slack-messenger](https://github.com/personium/personium-ex-slack-messenger)
 * [personium-ex-ew-services](https://github.com/personium/personium-ex-ew-services)
 
-## Installation
-
-### Install
+## Install the pre-built extensions
```
Comment (tochi-y) [inline ja/server-operator/setup_engine_extensions.md]:
```suggestion
## ビルド済Extensionのインストール  
```

File: ja/server-operator/setup_engine_extensions.md [hunk 2]
```diff
@@ -21,16 +19,19 @@ JARファイルをAPサーバに配備し、再起動してください。
 # systemctl restart tomcat
 ```
 
-### Update
-Engine Extensionをアップデートしたい場合は、JARファイルを上書きして、インストール時と同様に再起動を行ってください。
+## Build and install the latest extensions
```
Comment (tochi-y) [inline ja/server-operator/setup_engine_extensions.md]:
```suggestion
## 最新版Extensionのビルドとインストール  
```

File: ja/server-operator/setup_engine_extensions.md [hunk 3]
```diff
@@ -21,16 +19,19 @@ JARファイルをAPサーバに配備し、再起動してください。
 # systemctl restart tomcat
 ```
 
-### Update
-Engine Extensionをアップデートしたい場合は、JARファイルを上書きして、インストール時と同様に再起動を行ってください。
+## Build and install the latest extensions  
+最新のEngine ExtensionをbuildしてAPサーバーに配備したい場合は下記を実施してください。
 
-## JARファイルの作成
+### Pre-requisite (Maven)
```
Comment (tochi-y) [inline ja/server-operator/setup_engine_extensions.md]:
```suggestion
### 前提条件 (Maven)  
```

File: ja/server-operator/setup_engine_extensions.md [hunk 4]
```diff
@@ -21,16 +19,19 @@ JARファイルをAPサーバに配備し、再起動してください。
...
-Engine Extensionをアップデートしたい場合は、JARファイルを上書きして、インストール時と同様に再起動を行ってください。
+## Build and install the latest extensions  
+最新のEngine ExtensionをbuildしてAPサーバーに配備したい場合は下記を実施してください。
 
-## JARファイルの作成
+### Pre-requisite (Maven)  
 Engine ExtensionのビルドにはMavenが必要です。  
 http://maven.apache.org/install.html を参考にインストールを行ってください。  
 
+### Procedures
```
Comment (tochi-y) [inline ja/server-operator/setup_engine_extensions.md]:
```suggestion
### 手順  
```

**Review by tochi-y — COMMENTED** [review]
Thanks. The procedures are perfect. Only translations I suggested.

**Review by tochi-y — APPROVED** [review]
Thanks!

**Pull request merged**